lighthouse-core/audits/accessibility/list.js

```diff
@@ -22,6 +22,8 @@ class List extends AxeAudit {
       name: 'list',
       description: 'Lists contain only `<li>` elements and script supporting elements ' +
           '(`<script>` and `<template>`).',
+      fallbackDescription: 'Lists do not contain only `<li>` elements and script ' +
+          'supporting elements (`<script>` and `<template>`).',
```

Comment:
Side note: In this iteration I see that we're just inversing the description. But I think sometimes that leads to clunky sentences. We can make the fallbacks more readable by rephrasing them:

E.g. this one could be changed to `Lists contain invalid elements`

I'm not asking you to do this now, just noting it